### PR TITLE
show toast when running scheduler first time

### LINKED ISSUE #1816: Show toast when running scheduler for first time.
**Is your feature request related to a problem? Please describe.**
It's very annoying when you see an empty my time page for x seconds when you open the app for the first time.

**Describe the solution you'd like**
Show a toast when there is no schedule jn cache: "Automagical scheduling of your goals in progress..."

**Describe alternatives you've considered**
1. Start calculating async on the choose language page already. By the time you get to mytime the result is ready. Possible? More complicated?
2. Precache the default schedule with the app - skipping the first calculation alltogether.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #1816

### CODE REVIEW

**Review by tijlleenders — CHANGES_REQUESTED** [review]
Upon review, the Toast approach is not the right one. It will show the Toast while the goals have already loaded.  

A better way is to have a default DOM element that shows 'Automatic scheduling of your goals in progress...' - and then replace that with the schedule once it's available.  

So instead of showing the empty list of days, only show the message in the middle of the screen.

**Review by tijlleenders — CHANGES_REQUESTED** [review]
Yup, this is nicer.

Can you remove the italics (makes it harder to read)?
Also, please shorten the sentence to "Auto-scheduling...".

Then we're good to go!

**Review by tijlleenders — APPROVED** [review]
Perfect!

File: src/pages/MyTimePage/MyTimePage.scss [hunk 1]
```diff
@@ -49,3 +49,12 @@
   left: 4%;
   border-radius: 8px;
 }
+
+.scheduling-message {
```
Comment (Tushar-4781) [inline src/pages/MyTimePage/MyTimePage.scss:53]: please use a custom class from short.scss for the font weight and font size prop

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Comment by tijlleenders** [comment]
@Tushar-4781 *bump*

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**